src/test/java/uk/gov/companieshouse/chd/order/api/ChdOrderApiApplicationTests.java

```diff
@@ -1,13 +1,195 @@
 package uk.gov.companieshouse.chd.order.api;
 
+import org.junit.Rule;
```

Comment:
You don't have to do this one, but you could also use parameterized tests. https://www.tutorialspoint.com/junit/junit_parameterized_test.htm

Reply:
Nice didn't know about this, will have to spend some time working it out but a great suggestion to look into for the future